src/imdb_utils.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import kagglehub
+import os
+import shutil
+
+resources_path = "../resources"
+file_name = "movie_metadata.csv"
+kaggle_data = "carolzhangdc/imdb-5000-movie-dataset"
+
+def aquireIMDbDataFrame():
+    """
+    Aquires the IMDb data from a local file or from Kaggle if the file does not exist locally.
```

Comment:
Can we add  `Returns:` Section with `DataFrame: Pre-populated DataFrame object.`